public/css/style.css

```diff
@@ -0,0 +1,62 @@
+
+.comment-list .row {
+    margin-bottom: 0;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=5982252)